package/MDAnalysis/coordinates/memory.py

```diff
@@ -510,26 +512,11 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc', format=No
             of 'a', 'f', 'c' are allowed ie "fac" - return array
             where the shape is (frame, number of atoms,
             coordinates).
-        format : str (optional)
-            deprecated, equivalent to `order`
 
-        Note
-        ----
-        The `format` parameter name is used to mimic the
-        :class:`MDAnalysis.coordinates.DCD.timeseries` interface. It is
-        identical to the `order` parameter for :class:`MemoryReader`. In a
-        future version, `format` will be renamed to `order`.
 
-
-        .. deprecated:: 0.17.0
```

Comment:
IIRC, we leave in the old deprecation notices, sort of a history – or correct me...

Reply:
I'm not sure we've been consistent with this, (see: https://github.com/MDAnalysis/mdanalysis/pull/2494#discussion_r374805918). Personally, I'm always in favour of maximising history in docstrings, so I am more than willing to add this one back if you would like to.

Reply:
Thanks for showing that I have no idea what I said in the past. I am feeling very old  and senile now....

If we removed similar things recently then let's be consistent with the recent past. In any case, it's 1.0, who wants to look back? ;-)